hvplot/util.py

```diff
@@ -446,6 +446,16 @@ def is_xarray(data):
     return isinstance(data, (DataArray, Dataset))
 
 
+def is_lazy_data(data):
```

Comment:
```suggestion
def is_lazy_data(data):
    """ Check if data is lazy

    This checks if the datatype is Dask, Ibis, or Polars' LazyFrame.
    It is useful to avoid eager evaluation of the data.
    """
```